examples/cookbook/testing/unit/counter_app/test/group.dart

```diff
@@ -1,25 +1,21 @@
-import 'package:counter_app/counter.dart';
-import 'package:test/test.dart';
+import "package:counter_app/counter.dart";
```

Comment:
I'm curious about this change. I thought we had prefer single quotes lint rule in place. If not, we should add it.

Reply:
We do have the lint added, CI caught it.

Reply:
```suggestion
import 'package:counter_app/counter.dart';
```